app/controllers/api.js

```diff
@@ -6,85 +6,88 @@ var ErrorReport = require('../models/errorReport');
 var API = new Object();
 
 API.checkOut = function (req, res) {
-    //Get string data from card and send to other API - get studentID back
     var cardString = req.body.cardString;
     var dockID = req.body.dockID;
     var bikeID = req.body.bikeID;
-
-    //Look up if student currently has another bike out
-    Dock.findOne({ dockID: dockID}, function(err, dock) {
+	// TODO - check for already checked out bike
+	var validTransaction = true;
```

Comment:
validTransaction never changes from `true`, is that just a placeholder for later when the TODO is implemented?